packages/core/src/application.ts

```diff
@@ -47,21 +47,22 @@ export class Application extends Context {
       }
     }
 
-    this._bindSequence();
+    this._bindSequence(options && options.sequence);
 
     this.handleHttp = (req: ServerRequest, res: ServerResponse) =>
       this._handleHttpRequest(req, res);
 
     this.bind('logError').to(this._logError.bind(this));
   }
 
-  protected _bindSequence(): void {
+  public sequence(sequence: Constructor<Sequence>) {
+    this._bindSequence(sequence);
+  }
+
+  protected _bindSequence(sequence?: Constructor<Sequence>): void {
     // TODO(bajtos, ritch, superkhau) figure out how to integrate this single
     // sequence with custom sequences contributed by components
-    const sequence = this.options && this.options.sequence ?
-      this.options.sequence :
-      Sequence;
-    this.bind('sequence').toClass(sequence);
+    this.bind('sequence').toClass(sequence || Sequence);
```

Comment:
I find it rather surprising that a method called `_bindSequence` has a fallback to the default `Sequence` class.  My proposal:

```ts
// inside the constructor
this._setupDefaultSequence(options);

public sequence(sequence: Constructor<Sequence>) {
  this.bind('sequence').toClass(sequence);
}

protected _setupDefaultSequence(options?: ApplicationOptions) {
  const sequence = this.options && this.options.sequence ?
    this.options.sequence :
    Sequence;
  this.sequence(sequence);
}
```

Alternatively, I think `_setupDefaultSequence` can be replaced by a single line:

```ts
this.sequence(options && options.sequence || Sequence);
```